content/github/collaborating-with-issues-and-pull-requests/about-pull-requests.md

```diff
@@ -53,6 +53,10 @@ When you create a pull request, you can choose to create a pull request that is
 
 {% data reusables.pull_requests.mark-ready-review %} You can convert a pull request to a draft at any time. For more information, see "[Changing the stage of a pull request](/articles/changing-the-stage-of-a-pull-request)."
 
+### Difference between commits on Compare page and Pull Request page
```

Comment:
### Further reading

- "[Pull request](/articles/github-glossary/#pull-request)" in the {% data variables.product.prodname_dotcom %} glossary